discovery-infra/junit_log_parser.py

```diff
@@ -83,6 +84,40 @@ def export_service_logs_to_junit_suites(source_dir: Path, report_dir: Path):
         f.write(xml_report)
 
 
+def get_event_test_case(event: dict):
+    event_type = event["type"]
+    if event_type.lower() in EXPORTED_EVENT_LEVELS:
+        log.info(f"Adding {event_type} event as test-case")
+        reason = event["reason"]
+        involved_object = event["involvedObject"]
+        obj = involved_object["kind"].lower() + "/" + involved_object["name"]
+        message = event["message"]
+        timestamp = event["firstTimestamp"]
+        test_case = TestCase(name=obj, classname="EVENT", category=reason, timestamp=timestamp)
+        test_case.failures.append(CaseFailure(message=message, output=message, type=event_type))
```

Comment:
this doesn't yet treats them as flakes, right?

Reply:
From a quick look it's doesn't look like there are a lot of failures on events as in the service log. If there will be more then expected we will treats them as flaky. WDYT?